SharpBucketTests/V1/EndPoints/ChangesetEndPointTests.cs

```diff
@@ -4,10 +4,11 @@
 using Shouldly;
 using System;
 using System.Linq;
+using Xunit;

```

Comment:
Why moving to Xunit ?
I know that the pull request has stayed without any response for a long time and that nunit was not well documented about their support of the new dotnetcore test platform, but it can still works (and is even in the VS 2017 templates now)

Since @MitjaBezensek start with Nunit I think he is more used by that framework
I'm now a contributor and will be active to maintain that repo, and I'm also more used to nunit than xunit

so let stay on nunit, no need to change that

Reply:
Yeah, I would not like to change it, especially since no reason was given for the change.

Reply:
Can’t remember why I changed it, it was so long ago. Think prob could not get nunit working with notnet standard. I will try to have a look at this pull request again at some point, but truth be told, I’ve moved on and had forgotten that this was here.

Reply:
I clearly understand that time has passed
Let me know if you struggle to find time for that

Reply:
Ta, sorry just not had a chance to get back to it